istioctl/pkg/validate/validate.go

```diff
@@ -276,25 +277,71 @@ func validateFiles(istioNamespace *string, defaultNamespace string, filenames []
 	var errs, err error
```

Comment:
```suggestion
	var errs error
```

Do we really need `err` (singular) AND the global `errs` (plural) to both be global to the closure funcs?

I think we can probably drop `err` here and just keep it locally scoped to each local func - only `errs` needs to be in the outer scope AFAICT.